test1.js

```diff
@@ -3,3 +3,9 @@ var foo = '' ;
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var fooBar = '' ;
+
+if(fooBar                 === "bar") {
```

Comment:
Codacy found an issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26135)
```suggestion
if(fooBar === "bar") {
```